homeassistant/components/webostv/media_player.py

```diff
@@ -316,6 +302,23 @@ def supported_features(self):
 
         return supported
 
+    @property
+    def device_info(self):
+        """Return device information."""
+        if self._client.system_info is None and self.state == STATE_OFF:
+            return {
+                "identifiers": {(DOMAIN, self._unique_id)},
+            }
+        maj_v = self._client.software_info.get("major_ver")
+        min_v = self._client.software_info.get("minor_ver")
+        return {
+            "identifiers": {(DOMAIN, self._unique_id)},
+            "manufacturer": "LG",
+            "name": self._name,
+            "model": self._client.system_info.get("modelName"),
```

Comment:
Usually we exclude key/value pairs when the value will be `None`. Same for version.